### ISSUE TITLE
Verse highlight disappeared...

### ISSUE THREAD

**Issue by bvahedy** [AndBible/and-bible#2453]
Running 673.1 alpha build:

Tapping on the verse does not highlight the verse. Also when speak is on and the option to follow the speak verse is turned on, the verse being read is not highlighted.

Ticket ref:
https://support.andbible.org/scp/tickets.php?id=1754

**Comment by tuomas2** [comment]
I can't reproduce this issue

**Comment by bvahedy** [comment]
Both this and #2457 happen in the latest alpha build. Could this be a webview issue?
But i have the latest webview for Android 12. I know you are using Android 13.

**Comment by bvahedy** [comment]
I installed the 673 production and that works fine except it won't let me restore the database I saved from the alpha build. Any ideas why it won't restore the database from the alpha build?

**Comment by timbze** [comment]
> I installed the 673 production and that works fine except it won't let me restore the database I saved from the alpha build. Any ideas why it won't restore the database from the alpha build?

Yes, alpha build database can no longer be downgraded to production because it's database has schema changes.

I also did not have this issue in alpha... not sure what was going on. Can you try a new alpha install (without you db) and see if you have the issue?

**Comment by bvahedy** [comment]
I did a fresh install and still the same, so I went back to 667.1 alpha build and that works fine. Something happened after 667.1 that is causing this.

**Comment by bvahedy** [comment]
Well, in 667 the app would crash when you attempt to go into the bookmark list from the menu. Trying to fix that most likely has something to do with this issue, but that's just my guess.

**Comment by tuomas2** [comment]
try with alpha-673.3

**Comment by bvahedy** [comment]
I installed 673.4, I can see bookmark highlight now, but when I tap on a verse it does not highlight the verse, also the speak verse highlight still doesn't work, it reads the text but does not highlight the verse as it reads it.

**Comment by tuomas2** [comment]
Does highlight work OK when there's zero customization made, i.e. fresh install without any special documents installed (just KJV for example), no fonts, no addons, etc..?

**Comment by bvahedy** [comment]
Ok, I just did a fresh install and the verse highlight works with just kjv installed. Then I restored my bookmark database and tried it and the highlight still works. I added a few of  my Bible module and that still works.
But why would a  prior alpha builds work with all my Bible addons and not the recent alpha build?
at this point I would have to go through and install everything individually to see what causes this. Still there is no guarantee I would find the issue.
